ibm_mq/datadog_checks/ibm_mq/collectors/channel_metric_collector.py

```diff
@@ -171,6 +171,17 @@ def _submit_metrics_from_properties(self, channel_info, channel_name, metrics_ma
             if pymqi_type not in channel_info:
                 self.log.debug("metric '%s' not found in channel: %s", metric_name, channel_name)
                 continue
+
+            # Special handling for connection metric
+            if metric_name == 'conns':
+                connection_name = to_string(channel_info[pymqi_type]).strip()
+                if not connection_name:
+                    continue
+                connection_tags = tags + ["connection:{}".format(connection_name)]
```

Comment:
The connection tag, is there any chance it can be unbounded? Or what does a connection look like?